src/api/controllers/getAuthorsController.js

```diff
@@ -0,0 +1,45 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+import generatePaginationDetails from '../../helpers/generate.pagination.details';
+
+const { Article } = models;
+const { User } = models;
+/**
+ *
+ *
+ * @export
+ * @class getAuthors
+ */
+export default class getAuthors {
+  /**
+   *
+   *
+   * @static
+   * @param {array} req - request array
+   * @param {array} res - respond array
+   * @returns {array} response body
+   * @memberof getAuthors
+   */
+  static async authors(req, res) {
+    const { offset, limit } = req.query;
+    const paginatedUser = await User.findAndCountAll({
+      attributes: ['id', 'username', 'email'],
+      include: [
+        {
+          model: Article,
+          attributes: []
+        }
+      ],
+      offset,
```

Comment:
@Ramadhan0 set default values for limit and offset. See @nkpremices and @truestbyheart for context on this feedback.